docs/install.md

```diff
@@ -393,6 +401,9 @@ Select the following tabs for the installation description of your target.
 
         For the first boot up of your PC disconnect your USB stick from the target PC and power on the PC to login and execute the first boot procedure.
 
+        If you do not have a wired LAN connection you have to change the network settings matching to your environment (files `\boot\dietpi.txt` and `\boot\dietpi-wifi.txt`).  
```

Comment:
I was stumbling upon the negation and was wondering why for LAN connection something would need to be pre-configured. What about:
```
If you use a WiFi connection you have to ...
```
?

Reply:
Changed.